specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2021-10-01-preview/scanningService.json

```diff
@@ -486,6 +486,181 @@
         }
       }
     },
+    "/credentials/{credentialName}": {
+      "get": {
+        "tags": [
+          "Credential"
+        ],
+        "description": "Gets credential information",
+        "operationId": "Credential_Get",
```

Comment:
Also you might want to rethink on "Credential" in operationId. It likely too generic for SDK.

Reply:
I tried doing this today and encountered a weird issue. "Credentials" is a property used by autorest within autogenerated constructor. That is preventing me to use another property with the same name, while generating SDK. I posted a question [here](https://stackoverflow.microsoft.com/questions/294939) explaining the problem. If no one responds in next few days, I will ask the same in azure/autorest github repo. Since this is not a blocker my current change, I'm leaving it at `Credential_Get` for now.